2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Configure stats pid to 1666339
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from /home/anwer/.config/wandb/settings
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from /12tb_dsk3/SEIR_model/src/wandb/settings
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'sweeps-SEIR_Skåne_Kronoberg', 'entity': 'danish2562022', 'root_dir': '/12tb_dsk3/SEIR_model/src', 'run_id': 'ttooextq', 'sweep_param_path': '/12tb_dsk3/SEIR_model/src/wandb/sweep-yw5aij6r/config-ttooextq.yaml', 'sweep_id': 'yw5aij6r'}
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:_log_setup():529] Logging user logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250529_095856-ttooextq/logs/debug.log
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:_log_setup():530] Logging internal logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250529_095856-ttooextq/logs/debug-internal.log
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:_jupyter_setup():475] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7a5e7a12b110>
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():569] calling init triggers
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():576] wandb.init called with sweep_config: {'ah0': 5.021006698105129, 'ah1': 0.12037981931182153, 'bh0': 0.705430494634105, 'bh1': 1.7208803751812174, 'c': 1.0552475912635784}
config: {}
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():619] starting backend
2025-05-29 09:58:56,303 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():623] setting up manager
2025-05-29 09:58:56,305 INFO    Thread-3194 (_run_job):1666339 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-29 09:58:56,305 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():631] backend started and connected
2025-05-29 09:58:56,310 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_config_callback():1382] config_cb None None {'ah0': 5.021006698105129, 'ah1': 0.12037981931182153, 'bh0': 0.705430494634105, 'bh1': 1.7208803751812174, 'c': 1.0552475912635784}
2025-05-29 09:58:56,310 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2025-05-29 09:58:56,310 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-29 09:58:56,310 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():720] updated telemetry
2025-05-29 09:58:56,313 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-29 09:58:56,721 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_on_init():2402] communicating current version
2025-05-29 09:58:56,796 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-29 09:58:56,796 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():804] starting run threads in backend
2025-05-29 09:58:57,568 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_console_start():2380] atexit reg
2025-05-29 09:58:57,568 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-05-29 09:58:57,568 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-05-29 09:58:57,568 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_redirect():2325] Redirects installed.
2025-05-29 09:58:57,569 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:init():847] run started, returning control to user process
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_finish():2109] finishing run danish2562022/sweeps-SEIR_Skåne_Kronoberg/ttooextq
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [jupyter.py:save_history():444] not saving jupyter history
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [wandb_init.py:_jupyter_teardown():457] cleaning up jupyter logic
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2025-05-29 09:58:57,642 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_restore():2332] restore
2025-05-29 09:58:57,643 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_restore():2338] restore done
2025-05-29 09:59:00,775 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2025-05-29 09:59:00,776 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2025-05-29 09:59:00,778 INFO    Thread-3194 (_run_job):1666339 [wandb_run.py:_footer_sync_info():3950] logging synced files
